docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -124,4 +124,4 @@ Please note: Fidesops will filter out any sensitive configuration variables. The
 - `TASK_RETRY_DELAY`
 - `TASK_RETRY_BACKOFF`
 
-For more information please see the [api docs](/docs#/Config).
+For more information please see the [api docs](/api#operations-tag-Config).
```

Comment:
It looks like masking strategies guides still have links formatted like this too https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/guides/masking_strategies.md

Reply:
We also have this `/docs#` style formatting for links in several code comments in the tutorial and the quickstart - they're not live links, but perhaps we should remove them -

Reply:
Ah awesome callout, i'll remove these too!